Makefile

```diff
@@ -0,0 +1,22 @@
+LOCAL_BIN:=$(CURDIR)/bin
+
+install-deps:
+	GOBIN=$(LOCAL_BIN) go install google.golang.org/protobuf/cmd/protoc-gen-go@latest
+	GOBIN=$(LOCAL_BIN) go install -mod=mod google.golang.org/grpc/cmd/protoc-gen-go-grpc@latest
+
+get-deps:
+	go get -u google.golang.org/protobuf/cmd/protoc-gen-go
+	go get -u google.golang.org/grpc/cmd/protoc-gen-go-grpc
+
+
+generate:
+	make generate-user-api
+
+generate-user-api:
+	mkdir -p pkg/
+	protoc --proto_path=/Users/assultuss/Basic/GolandProjects/messenger-console/auth/api/v1/ \
```

Comment:
Nobody will be able to run this task because it is tightly coupled onto your directory structure 
`/Users/assultuss/Basic/GolandProjects/...`